### PR TITLE
[sipify] creates links in seealso

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: scripts/sipify.pl [hunk 1]
```diff
@@ -133,8 +133,17 @@ sub processDoxygenLine {
...
         return ".. versionadded:: $1\n";
     }
-    if ( $line =~ m/\\see (.*)/ ) {
-        return ".. seealso:: $1\n";
+    if ( $line =~ m/\\see *(.*)/ ) {
+        my $seealso = $1;
+        if (  $1 =~ m/^Qgs[A-Z]/ ) {
+            return ".. seealso:: :py:class:`$seealso`\n";
+        }
+        elsif (  $1 =~ m/^[a-z]/ ) {
```
Comment (nyalldawson) [inline scripts/sipify.pl]: I suspect you should need to detect that there's no '::' here

File: scripts/sipify.pl [hunk 2]
```diff
@@ -133,8 +133,17 @@ sub processDoxygenLine {
     if ( $line =~ m/\\since .*?([\d\.]+)/i ) {
         return ".. versionadded:: $1\n";
     }
-    if ( $line =~ m/\\see (.*)/ ) {
-        return ".. seealso:: $1\n";
+    if ( $line =~ m/\\see *(.*)/ ) {
+        my $seealso = $1;
+        if (  $1 =~ m/^Qgs[A-Z]/ ) {
```
Comment (nyalldawson) [inline scripts/sipify.pl]: i think this one also should check for `.`s, and if so use the func type link

**Comment by nyalldawson** [comment]
cool!

File: scripts/sipify.pl (hunk 1 above)
Comment (3nids) [reply scripts/sipify.pl]: good catch

File: scripts/sipify.pl (hunk 2 above)
Comment (3nids) [reply scripts/sipify.pl]: indeed

**Comment by NathanW2** [comment]
Ohhhh super nice!

**Comment by nyalldawson** [comment]
*thinking aloud* wonder if we could make prepare commit also standardise the doxygen see also style... e.g. consistent use of brackets or not...

**Comment by 3nids** [comment]
@nyalldawson yes we'll need something like this, Sphinx seems to be a bit more restrictive about the order of the element in the documentation

**Pull request merged**